application-logging/esm-logs-in-context/app/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# syntax=docker/dockerfile:1
+
+# Comments are provided throughout this file to help you get started.
+# If you need more help, visit the Dockerfile reference guide at
+# https://docs.docker.com/engine/reference/builder/
+
+ARG NODE_VERSION=18.17.0
+
+FROM node:${NODE_VERSION}-alpine
+
+# Use production node environment by default.
+ENV NODE_ENV production
+
+
+WORKDIR /app
+
+# Download dependencies as a separate step to take advantage of Docker's caching.
+# Leverage a cache mount to /root/.npm to speed up subsequent builds.
+# Leverage a bind mounts to package.json and package-lock.json to avoid having to copy them into
+# into this layer.
+RUN --mount=type=bind,source=package.json,target=package.json \
+    --mount=type=bind,source=package-lock.json,target=package-lock.json \
+    --mount=type=cache,target=/root/.npm \
+    npm ci --omit=dev
+
+# Run the application as a non-root user.
+USER node
+
+# Copy the rest of the source files into the image.
+COPY .. .
+
+# Expose the port that the application listens on.
+EXPOSE 3000
+
+# Run the application.
+CMD node --experimental-loader newrelic/esm-loader.mjs -r newrelic index.js  > ./logs/logs-in-context-esm.log
```

Comment:
What is the purpose of redirecting the logs into a file? This will prevent standard Docker tooling from being able to present and manage the logs.

Reply:
yea, I was trying to find the logs and saw this, I wouldn't do this

Reply:
That was ported over from one of our other examples. Happy to take that out.

Reply:
yea that's because that example uses a log forwarder and we only wanted to forward logs generated by logging lib. this is a diff use case